content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md

```diff
@@ -19,18 +19,95 @@ Please contact R3 if you are interested in joining the Beta program.
 
 This section describes the new features in Corda 5.0 Beta 2.0.
 
+### Cluster Configuration
 
+#### Mutual TLS
+It is now possible to configure Corda cluster gateways to connect with each other via [mutual TLS]({{< relref "../operating/mutual-tls-connections.md" >}}).
+
+#### Messaging Maximum Size
+A new `messaging` [configuration field]({{< relref "../operating/configuration/config-overview.md" >}}), `maxAllowedMessageSize`, enables you to specify the maximum size of a message sent from a Corda worker to the message bus. Corda breaks messages that exceed this size into smaller messages before sending.
+The `maxAllowedMessageSize` value must be lower than the maximum message size configured on the message bus itself, for example, Kafka. If the Corda configuration value is set to a value higher than that of the message bus, the messages will not send.
+
+#### Registration Approval
+A network operator can now configure a membership group to require that the operator must manually approve (or decline) member registration requests. Requests satisfying specific criteria require manual approval, while others are approved automatically. The operator can also pre-authenticate specific members, allowing them to bypass the standard approval rules defined for the group. The operator can further configure pre-authentication to specify that certain changes to the member’s context must be manually reviewed. For more information, see [Manual Registration Approval]({{< relref "../operating/registration-approval.md" >}}).
```

Comment:
```suggestion
A network operator can now configure a membership group so that the operator is required to manually approve (or decline) member registration requests. Requests satisfying specific criteria require manual approval, while others are approved automatically. The operator can also pre-authenticate specific members, allowing them to bypass the standard approval rules defined for the group. The operator can further configure pre-authentication to specify that certain changes to the member’s context must be manually reviewed. For more information, see [Manual Registration Approval]({{< relref "../operating/registration-approval.md" >}}).
```